### DISCUSSION TITLE
SNOTEL stations missing snow depth and water equivalent fields

### DISCUSSION THREAD

**Discussion by bcbeta** [weather-gov/api#748] [General]
For example:
https://api.weather.gov/stations/BTSC2/observations
The snotel data for this station is here- https://wcc.sc.egov.usda.gov/nwcc/site?sitenum=335

**Comment by dwhitemv25** [comment]
This API only supports a subset of sensors commonly used for terrestrial observation, i.e. temperature, humidity, wind speed, etc. If you need specialty sensor output you'll have to query the provider directly (or MADIS if its in there, but I'd just go straight to the source for SNOTEL). There is an API available at https://wcc.sc.egov.usda.gov/awdbRestApi/swagger-ui/index.html.

EDIT: For snow depth and SWE in particular you _really_ want to look at the trend data as those sensors are notoriously glitchy.

**Reply by bcbeta** [reply]
Thanks, I am already using the SNOTEL API. I was hoping to consolidate all data queries using this API but doesn't seem like that will work out.
